docs/development/core/server/kibana-plugin-core-server.validateobject.md

```diff
@@ -0,0 +1,22 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [kibana-plugin-core-server](./kibana-plugin-core-server.md) &gt; [validateObject](./kibana-plugin-core-server.validateobject.md)
+
+## validateObject() function
```

Comment:
It might make sense to add an explanatory sentence to the documentation of this helper